pkgs/stdenv/linux/make-bootstrap-tools.nix

```diff
@@ -135,13 +147,14 @@ rec {
         for i in $out/bin/* $out/libexec/gcc/*/*/*; do
             if test -x $i -a ! -L $i; then
                 chmod +w $i
-                strip -s $i || true
+                $STRIP -s $i || true
             fi
         done
 
         nuke-refs $out/bin/*
         nuke-refs $out/lib/*
         nuke-refs $out/libexec/gcc/*/*/*
+        nuke-refs $out/lib/gcc/*/*/*
```

Comment:
This was only recently needed by cross, for reasons unknown. Should be harmless for native?